.github/workflows/ipc_fuzzer.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+# For the actual fuzzer see tools/oss-fuzz/README.
+# also see
+# See https://google.github.io/oss-fuzz/getting-started/continuous-integration/
+#
+# Build and run fuzzer for 5s to check if it runs properly. If it consistently
+# fails in under 5s you probably did something wrong
+
+name: IPC fuzzer compile test
+
+# 'workflow_dispatch' allows running this workflow manually from the
+# 'Actions' tab
+# yamllint disable-line rule:truthy
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  ipc-fuzzer-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build Fuzzers
+        uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+        with:
+          oss-fuzz-project-name: 'sound-open-firmware'
+
+      - name: Run Fuzzers
+        uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
+        with:
+          oss-fuzz-project-name: 'sound-open-firmware'
+          language: c
+          fuzz-seconds: 5
```

Comment:
Fuzzer newbie question sorry: why 5 seconds and not 1 or 20 seconds? Would this deserve a comment?

Are there in the logs some numbers showing the extend of the test coverage in that time?
https://github.com/thesofproject/sof/runs/2605452860?check_suite_focus=true

Speaking of the output, I assume the following "Errors" are not actually errors?
```
021-05-17 23:02:16,179 - root - DEBUG - Stderr of command 'docker rmi -f gcr.io/oss-fuzz/sound-open-firmware gcr.io/oss-fuzz-base/base-runner gcr.io/oss-fuzz-base/msan-libs-builder' is Error: No such image: gcr.io/oss-fuzz-base/msan-libs-builder


2021-05-17 23:02:31,613 - root - ERROR - Fuzzer fuzz_ipc timed out, ending fuzzing
```

Reply:
Typically fuzzers run for much longer, i think on oss-fuzz the default is somewhere around 300-600. Hence the comment at the top explaining that its just a quick check. The longer we run the more likely we are to hit a legitimate error that oss-fuzz should find rather than breaking the CI for something likely not related to the actual change being tested. Plus setting up log for collection for a proper crash is a lot more work.

And yes those are non-errors. I'm guessing the fuzzer just setup `alarm` and that is the signal handler logging it. Confirmed by the fact that the process exited cleanly

Reply:
Thanks, should have paid more attention to the comment at the top.